airbyte-integrations/connectors/source-aha/source_aha/spec.yaml

```diff
@@ -0,0 +1,18 @@
+documentationUrl: https://docs.airbyte.com/integrations/sources/aha
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Aha Spec
+  type: object
+  required:
+    - api_key
+    - url
+  additionalProperties: true
+  properties:
+    api_key:
+      type: string
+      description: API Key
+      title: API BEARER TOKEN
```

Comment:
Please capitalize the title correctly: API Bearer Token

Reply:
Same for the other parameter.

Reply:
Updated the title as suggested